src/njoy21/input/ACER/Card7/Thin1.hpp

```diff
@@ -0,0 +1,20 @@
+struct Thin1 {
+  using Value_t = double;
```

Comment:
The manual states (and I believe is correct) that the units are in eV. This seems to be natural because it is in the units of ENDF (eV) not ACE (MeV).